doc/configuration.rst

```diff
@@ -24,6 +24,7 @@ Pytition specific settings
 
 .. autodata:: pytition.settings.base.USE_MAIL_QUEUE
 .. autodata:: pytition.settings.base.ALLOW_REGISTER
+.. autodata:: pytition.settings.base.DEFAULT_NOREPLY_MAIL
```

Comment:
I think this is already merged in `master`.
Could you rebase your commit on `origin/master` and update the pull request?